src/gcs.rs

```diff
@@ -640,23 +642,64 @@ mod tests {
     async fn get_private_object() {
         init();
 
+        // define the private object
         let private_bucket = test_bucket();
         let filename = "get_private_object.txt";
 
+        // generate test data of a few kB
+        let file_length = 4096;
+        let rand_string: String = thread_rng()
+            .sample_iter(&Alphanumeric)
+            .take(file_length)
+            .map(char::from)
+            .collect();
+        // upload the test data
+        let client = &new_client();
```

Comment:
Oh, I meant to take all this logic and make a little test helper (or even dead code module function) for the "make a client, start an upload, append, finalize")

Reply:
Done.